Registry/RegistryParse.py

```diff
@@ -322,15 +380,26 @@ def checksum(self):
 
     def validate_checksum(self):
         """
-        Is the file checksum valid.
+        Is the file checksum valid?
         """
         return self.calculate_checksum() == self.checksum()
 
-    def validate(self):
+    def recovery_required(self):
         """
-        Is the file checksum and sequence valid.
+        Are the file checksum and sequence numbers valid?
+        Return a tuple with two boolean values:
+          - the first one is True when the REGF block recovery is required,
+          - the second one is True when data recovery is required.
         """
-        return self.validate_checksum() and self.validate_sequence_numbers()
+        if not self.validate_checksum():
+            # Header is invalid, this also implies data recovery
+            return (True, True)
```

Comment:
Recommend using a `collections.namedtuple` here, rather an this implicit structure. `namedtuples` have approximately the same performance characteristics, yet are more self-documenting.